spec/features/admin_claim_approve_spec.rb

```diff
@@ -27,6 +27,13 @@
 
         expect(page).to have_content("Claim has been approved successfully")
         expect(page).to_not have_content(claim_to_approve.reference)
+
+        expect(ActionMailer::DeliveryJob).to have_been_enqueued.with(
```

Comment:
Let's do a more end-to-end check here and check deliveries to the person with the expected subject or something.